inventory/environ.py

```diff
@@ -250,6 +250,42 @@ def merge_dict(dict1, dict2, path=None):
             dict1[key] = dict2[key]
     return dict1
 
+def mergeDefaultSplunkVariables(vars_scope, url):
```

Comment:
Can you add documentation on the intent of this merge?  Especially when it comes to variable precedence.

Reply:
@mikedickey I'm going to merge this, please add a docstring for this later.